flytekit/configuration/internal.py

```diff
@@ -64,6 +64,13 @@ class Credentials(object):
     This command is executed to return a token using an external process.
     """
 
+    PROXY_COMMAND = ConfigEntry(
```

Comment:
TODO: do we need the legacy config entry or do we only support this in yaml config?

Reply:
i think this is the correct declaration.  Let's keep it.  But what is this?  what does this do?

Reply:
Added an explanation in this PR's description.